### ISSUE TITLE
[BUG] Projects is unusuable for me on Windows 11.

### ISSUE THREAD

**Issue by DudeThatsErin** [marcusolsson/obsidian-projects#609]
### What happened?

I just did a fresh install of my Windows 11 installation which meant I re-cloned my Obsidian vault to my PC. So everything is from the clone.

For some reason Projects keeps adding these weird frontmatter items to my pages.
Only thing in my template is this: (short template)
```
---
date applied: 2023-07-12
last update date: 2023-07-12
link: "[]()"
status: ""
---
Job Titles: 
- 1
- 2
- 3
 
> [!summary] Contact Person
> Information on Contact person

> [!updated] Last Updated on XX/XX
> Information
# Job Info
```
So you can see 2 date fields, a rich text field and a regular text field.

My pages keep getting things like this:
```
---
"\r\"\\r\\\"\\\\rlink\\\"\"": "[site](https://www.strazcenter.org/)"
status: ""
---
```
You can see here: 
<img width="1920" alt="Obsidian_K3dWE5ALbt" src="https://github.com/marcusolsson/obsidian-projects/assets/2681022/230a2838-b731-42b4-afc8-8e35c19c85d4">
and here:
![image](https://github.com/marcusolsson/obsidian-projects/assets/2681022/e0114260-d7b8-4f47-94fa-6eac8143365c)
Second picture is after re-installing the plugin.

Also, horizontal scrolling with my MX Master 3 mouse (has a horizontal scroll wheel) no longer works. I have options+ installed and the scroll wheel is set to scroll horizontally no matter what app and it works anywhere else.

### What did you expect to happen?

Additional frontmatter to not be added.

### How can we reproduce it (as minimally and precisely as possible)?

These are the steps I did... not sure if it was my exact steps or what that caused this:

- Clone an obsidian vault with projects already installed, enabled, and set up.
- Open Obsidian
- Open the projects plugin (with the button on the sidebar)
- Table view comes up with these weird columns added.
- Attempt to delete a column, more get added with a never ending cycle.

I have uninstalled Projects.
Rebooted Obsidian.
Re-installed Projects.
Re-setup projects.
Same issue.
I did that in this order.

Not sure what else to do but it is unusable for me. The horizontal scrolling issue is apparent in table and kanban view.

### Anything else we need to know?

This is sad because I love this plugin and I thought it would allow me to get more of a Notion-like experience in Obsidian but I can't use it if it won't work.

### Plugin version

1.16.0

### Obsidian version

1.3.5

### OS

Windows

**Comment by DudeThatsErin** [comment]
I thought it could have been the make.md plugin but I disabled that plugin and this is still happening.

**Comment by cliftonbartholomew** [comment]
I am getting this too. It only started happening today. The only change I did was delete and reclone my obsidian folder from github.

**Comment by cliftonbartholomew** [comment]
I would edit the issue name to be:
"[BUG] Projects is not correctly editing YAML in Windows 11." 

As this is a YAML editing issue.

**Comment by marcusolsson** [comment]
Sorry about that, that is unfortunate 🙏 I don't have an idea off the bat on what could cause this, and I don't have access to a Windows 11 PC, but I will investigate to see if I can find anything.

**Comment by Acylation** [comment]
I tested it on a win11 PC, it seems that the plugin works well with the template given above. Not sure if it's a system related issue.

@DudeThatsErin To help reproduce the problem, could you please provide **anonymized** details such as the plugin list, template config and  sample notes?

Also, @cliftonbartholomew, could you clarify which plugin you are referring to as the “yaml editor”?

**Comment by DudeThatsErin** [comment]
Sorry I meant to update this. Reinstalled the plugin after the latest update and it works fine now. Weirdly enough. Not sure what caused it.

**Comment by Acylation** [comment]
Got it, glad that it works.

And about horizontal scrolling, I found some threads for you. It seems that the horizontal scrolling problem of the MX Master series occurs on electron-built applications, including Obsidian. And some suggest it may be caused by the Logi Options+. Here are some potential solutions:

- **Test horizontal scroll wheel outside of Projects.** For example, a zoomed embed PDF, or an Obsidian canvas. I tested that, in all cases, the middle wheel with the `shift` key appended is always available. This means to make sure that this is not an issue of the Projects plugin, but of the Obsidian/Electron-built applications.
- **Disable hardware acceleration.** You can find the option in `Settings→About→Advanced`. However, this may cause serious performance problems. This works for some VSCode users, but some also reported that this may not be of help. 
- **Check Logi Options+ configuration.** Some users regard it as a Logi Options+ problem, the team hard-coded common applications they need to support. You can try to disable the Options+, or assign a specified config for Obsidian. (Even change the `.exe` file name to `Chrome.exe`)
- **Use the older Logitech Option instead.** Some claim that their problem is solved by falling back to Logitech Options to manage, rather than using Logi Options+

References:
- [Ban hardware acceleration](https://github.com/electron/electron/issues/20874#issuecomment-569559324)
- [Uninstall Options+](https://github.com/electron/electron/issues/20874#issuecomment-976441443)
- [Run target application (like Obsidian) as Administrator](https://github.com/electron/electron/issues/20874#issuecomment-977191227)
- [Specify config in Options+ for Obsidian](https://github.com/electron/electron/issues/20874#issuecomment-1292473205)(Maybe you need to rename the `.exe` file)
- [Specify config in Options+](https://forum.vivaldi.net/topic/41537/logitech-smooth-scroll-extension-is-not-working-in-vivaldi/14)

Complaints on this:
- https://github.com/microsoft/vscode/issues/56964
- https://forum.obsidian.md/t/add-horizontal-scrolling-using-mousewheel-to-canvas/56392
- https://www.reddit.com/r/logitech/comments/vfe12n/issue_with_mx_master_3_horizontal_scroll_in/

In conclusion, the plugin does not need to take any action to adapt horizontal scrolling. I suggest that this issue can be close, but feel free to raise any questions.

**Comment by cliftonbartholomew** [comment]
> Also, @cliftonbartholomew, could you clarify which plugin you are referring to as the “yaml editor”?

I meant to say that the way the projects plugin is managing the YAML.

It is a separate issue though and I have just updated the projects plugin. I will log a new issue if it persists in the update.

I think you can mark this as resolved.

**Comment by marcusolsson** [comment]
Thanks for taking the time to update! (And thanks @Acylation for helping out with troubleshooting! 🙏 )
